pkg/server/registry/apigroups/acorn/events/strategy.go

```diff
@@ -187,7 +187,7 @@ type prefix string
 func (p prefix) matches(e apiv1.Event) bool {
 	return p.all() ||
 		strings.HasPrefix(e.Name, string(p)) ||
-		strings.HasPrefix(e.Source.String(), string(p))
+		(e.Resource != nil && strings.HasPrefix(e.Resource.String(), string(p)))
```

Comment:
Does this work?
```suggestion
		strings.HasPrefix(z.D(e.Resource).String(), string(p))
```

Reply:
This is actually the exact line that prompted me to add `z.D` (now `z.Dereference`).

Will update this while rebasing.